simulator/src/file/Open.js

```diff
@@ -0,0 +1,66 @@
+import load from '../data/load';
+import { generateSaveData } from '../data/save';
+
+const scopeSchema = ['layout', 'verilogMetadata', 'allNodes', 'id', 'name', 'restrictedCircuitElementsUsed', 'nodes'];
+const JSONSchema = ['name', 'timePeriod', 'clockEnabled', 'projectId', 'focussedCircuit', 'orderedTabs', 'scopes'];
+
+var circuitData = null;
+const GetDialogData = () => '<div><label for="CircuitDataFile">Choose file</label><div id="message-box"><i class="fas fa-plus"></i><br/>Browse files or Drag & Drop files here<div id="fileName">No file chosen!!</div><input style="background:none;" type="file" id="CircuitDataFile"/></div></div>';
+
+const ImportCircuitFiles = () => {
+    $('#ImportCircuitFilesDialog').empty();
+    $('#ImportCircuitFilesDialog').append(GetDialogData());
+    $('#ImportCircuitFilesDialog').dialog({
+        resizable: false,
+        close() {
+            if (circuitData) load(circuitData);
+        },
+        buttons: [
+            {
+                text: 'Close',
+                click() {
+                    $(this).dialog('close');
+                },
+            },
+        ],
+    });
+    $('#ImportCircuitFilesDialog').focus();
+
+    function ValidateData(circuitData) {
```

Comment:
validate function to validate JSON file data...

working:


https://user-images.githubusercontent.com/76155456/180723367-c3723aa5-e10e-47fa-9ab5-5723c4e110f9.mov

Reply:
Ah great. Just one small nitpick: Can you think of a way to make the error message pop out a little more? Right now it's easy to miss

Reply:
<img width="680" alt="Screenshot 2022-07-26 at 12 45 58 AM" src="https://user-images.githubusercontent.com/76155456/180856766-245d056d-6e00-4f45-ab64-d777c894040d.png">

this seems good?

Reply:
@pulkit-30 Sorry for a very late review. I would say not to bother the user with the exact error. Can we just make this a simple "Invalid file format" error?

I'll merge this PR (because it's complete and standing for too long). You can open another small PR for this change

Reply:
That's fine
I will do that in other PR 🚀 
Thanks 🤗